unit_tests/test_service.py

```diff
@@ -0,0 +1,13 @@
+import unittest.mock
+
+from src import *
+
+
+class TestService(unittest.TestCase):
+    def test_run(self) -> None:
```

Comment:
maybe call it "test_service" if we want to have multiple test cases for Service at the end

Reply:
in fact "test_runs" as the test name is supposed to say what service does: it runs...

will change and re-submit